lib/Jsrt/JsrtDebugUtils.cpp

```diff
@@ -389,6 +394,16 @@ void JsrtDebugUtils::AddPropertyToObject(Js::DynamicObject * object, JsrtDebugPr
     JsrtDebugUtils::AddVarPropertyToObject(object, propertyId, value, scriptContext);
 }
 
+bool JsrtDebugUtils::HasProperty(Js::DynamicObject * object, JsrtDebugPropertyId propertyId, Js::ScriptContext * scriptContext)
+{
+    const char16* propertyName = GetDebugPropertyName(propertyId);
+
+    const Js::PropertyRecord* propertyRecord;
+    scriptContext->GetOrAddPropertyRecord(propertyName, static_cast<int>(wcslen(propertyName)), &propertyRecord);
+
+    return !!Js::JavascriptOperators::HasProperty(object, propertyRecord->GetPropertyId());
```

Comment:
I was getting warnings about conversion from `BOOL` (`typedef int`) to `bool`.  I didn't see any other usages of `BOOL` in this file, so I wanted to keep the signature as is.  Is there a better way to "BOOLIFY" this?